README.md

```diff
@@ -21,11 +21,15 @@ You should have the following ready before beginning with any board:
 -   [Arduino IDE 1.6.8](https://www.arduino.cc/en/Main/Software)
 -   Install the `AzureIoTHub` library via the Arduino IDE Library Manager
 
+# Simple Sample Instructions
+
 ## MKR1000 or Zero + Wifi101
-1. Open the AzureIoTHub SAMD sample from the Arduino IDE File->Examples menu.
-2. Update Wifi SSID/Password in simplesample_http.ino 
+1. Open the `simplesample_http` example from the Arduino IDE File->Examples->AzureIoTHub menu.
+2. Install the `RTCZero` library from the Arduino IDE Library Manager.
```

Comment:
We are missing the `WiFi101` library. I can add this after merging if it is easier.